static/app/utils/replays/replayReader.tsx

```diff
@@ -133,4 +135,17 @@ export default class ReplayReader {
   getMemorySpans = () => {
     return this.memorySpans;
   };
+
+  isNetworkDetailsSetup = () => {
+    if (this._isDetailsSetup === undefined) {
+      // TODO(replay): there must be a better way
```

Comment:
😬

Reply:
https://github.com/getsentry/sentry-javascript/issues/8004 to followup

Reply:
See also https://github.com/getsentry/sentry-javascript/pull/8011/